resources/app.yaml

```diff
@@ -12,15 +12,17 @@ hooks:
       metadata:
         name: logging-app-bootstrap
       spec:
+        parallelism: 1
+        completions: 1
         template:
           metadata:
             name: logging-app-bootstrap
           spec:
             restartPolicy: OnFailure
             containers:
               - name: hook
-                image: quay.io/gravitational/debian-tall:0.0.1
-                command: ["/usr/local/bin/kubectl", "apply", "-f", "/var/lib/gravity/resources/resources.yaml"]
+                image: log-hook:0.0.1
+                args: ["bootstrap"]
```

Comment:
You need to fix the uninstall hook as well, otherwise it's broken with this change.